src/univariate/locationscale.jl

```diff
@@ -1,129 +1,140 @@
 """
-    LocationScale(μ,σ,ρ)
+    AffineDistribution(μ, σ, ρ)
 
-A location-scale transformed distribution with location parameter `μ`,
-scale parameter `σ`, and given univariate distribution `ρ`.
+A shifted and scaled (affinely transformed) version of `ρ`.
 
-If ``Z`` is a random variable with distribution `ρ`, then the distribution of the random
-variable
+If `Z` is a random variable with distribution `ρ`, then `AffineDistribution(μ, σ, ρ)` is
+the distribution of the random variable
 ```math
-X = μ + σ Z
+X = μ + σ * Z
```

Comment:
This is not a code block but an equation:
```suggestion
X = μ + σ Z
```